libs/blocks/chart/list.js

```diff
@@ -0,0 +1,69 @@
+import { propertyNameCI, propertyValueCI } from './utils.js';
+
+export function listChartData(json) {
+  const data = [];
+
+  if (json[':type'] !== 'multi-sheet') {
+    console.warn('List chart error: Add table sheet');
+    return data;
+  }
+
+  const tableKey = propertyNameCI(json, 'table');
+
+  if (tableKey) {
+    json[tableKey].data?.forEach((column) => {
+      data.push({
+        title: propertyValueCI(column, 'title'),
+        list: json[propertyValueCI(column, 'sheet')]?.data,
+        type: propertyValueCI(column, 'type'),
+      });
+    });
+  } else {
+    json[':names'].forEach((sheet) => {
+      data.push({
+        title: sheet,
+        list: json[sheet]?.data,
+      });
+    });
+  }
+
+  return data;
+}
+
+export const getListHtml = (data) => {
+  // ToDo: Replace firstList with carousel iterator MWPW-114550
+  const firstList = data[0];
+  const listType = firstList.type?.toLowerCase() === 'numbered' ? 'ol' : 'ul';
+
+  const listItems = firstList.list.reduce((prev, listItem) => {
+    const name = propertyValueCI(listItem, 'name') || '';
+    const extra = propertyValueCI(listItem, 'extra');
+    const image = propertyValueCI(listItem, 'image');
+    const alt = propertyValueCI(listItem, 'alt') || '';
```

Comment:
FYI: Since list chart doesn't seem to be using any of the header information for rendering headers could be converted to all lowercase. This wouldn't work for other charts that use headers for the legend.

Example:
```
function objKeysToLowerCase(obj) {
  return Object.keys(obj).reduce((newObj, key) => {
    newObj[key.toLowerCase()] = obj[key];
    return newObj;
  }, {});
}
```
You would have to run this on each sheet.
```
const firstList = data[0].map(objKeysToLowerCase);
```